README.md

```diff
@@ -31,7 +31,7 @@ See language-specific documentation:
 - [Java](clients/java/README.md)
 - [JavaScript](clients/js/README.md)
 - [PHP](clients/php/README.md)
-- [Ruby](clients/ruby/README.md)
```

Comment:
Keep the ruby link, and put go in alphabetical order

Reply:
@sffc fixed